api/pubsub/pubsub_test.go

```diff
@@ -155,13 +153,10 @@ func (s *PubSubIntegrationSuite) SetUpTest(c *gc.C) {
 	s.AddCleanup(func(*gc.C) { statePool.Close() })
 	config := testserver.DefaultServerConfig(c)
 	config.Hub = s.hub
-	info, server, httpServer := testserver.NewServerWithConfig(c, statePool, config)
-	httpServer.StartTLS()
-	s.AddCleanup(func(*gc.C) { httpServer.Close() })
-	s.server = server
```

Comment:
Was this stored but never used? (and thus fine to remove it here)

Reply:
That's right - the test didn't use it directly, we just need the API info to connect.